groovy/groovy.editor/nbproject/org-netbeans-modules-groovy-editor.sig

```diff
@@ -44,24 +43,19 @@ meth public java.lang.String toString()
 
 CLSS public org.codehaus.groovy.ast.ASTNode
 cons public init()
-meth public <%0 extends java.lang.Object> {%%0} getNodeMetaData(java.lang.Object)
-meth public boolean equals(java.lang.Object)
+intf org.codehaus.groovy.ast.NodeMetaDataHandler
 meth public int getColumnNumber()
 meth public int getLastColumnNumber()
 meth public int getLastLineNumber()
 meth public int getLineNumber()
-meth public int hashCode()
-meth public java.lang.Object putNodeMetaData(java.lang.Object,java.lang.Object)
 meth public java.lang.String getText()
-meth public java.util.Map<?,?> getNodeMetaData()
-meth public org.codehaus.groovy.util.ListHashMap getMetaDataMap()
+meth public java.util.Map<?,?> getMetaDataMap()
 meth public void copyNodeMetaData(org.codehaus.groovy.ast.ASTNode)
-meth public void removeNodeMetaData(java.lang.Object)
 meth public void setColumnNumber(int)
 meth public void setLastColumnNumber(int)
 meth public void setLastLineNumber(int)
 meth public void setLineNumber(int)
-meth public void setNodeMetaData(java.lang.Object,java.lang.Object)
+meth public void setMetaDataMap(java.util.Map<?,?>)
```

Comment:
3rd party library has changed incompatibly. Hard to do something with it, except increasing release version. Increasing release version is a drastical change, but this is clearly incompatible, so I suggest to try: change the module to `/3` - hopefully not many modules depend on `groovy.editor`...

Reply:
Done.